core/modules/user/user.admin.inc

```diff
@@ -500,6 +509,13 @@ function user_admin_permissions_submit($form, &$form_state) {
 function user_admin_roles($form, $form_state) {
   $system_config = config('system.core');
 
+  $options = array(0 => t('disabled'));
+  $options += user_roles();
+  // Do not allow the anonymous or authenticated roles to be set as the
+  // administrator.
+  unset($options[BACKDROP_ANONYMOUS_ROLE]);
+  unset($options[BACKDROP_AUTHENTICATED_ROLE]);
+
```

Comment:
Fix the select, so that the "disabled" option is at the top of the list. See next comment...